### PR TITLE
log git and docker progress

### PR DESCRIPTION
**Pull request by soulseen**
Signed-off-by: soulseen <sunzhu@yunify.com>

### CODE REVIEW

**Comment by soulseen** [comment]
@runzexia @magicsong

**Comment by soulseen** [comment]
@runzexia

File: pkg/docker/docker.go [hunk 1]
```diff
@@ -581,7 +581,9 @@ func (d *stiDocker) PullImage(name string) (*api.Image, error) {
 					return msg.Error
 				}
 				if msg.Progress != nil {
-					glog.V(4).Infof("pulling image %s: %s", name, msg.Progress.String())
+					if msg.Progress.Current != 0 {
+						glog.V(4).Infof("pulling image %s: %s", name, msg.Progress.String())
```
Comment (runzexia) [inline pkg/docker/docker.go]: should glog.V(0)

File: pkg/docker/docker.go [hunk 2]
```diff
@@ -668,8 +670,11 @@ func (d *stiDocker) PushImage(name string) error {
 				if msg.Error != nil {
 					return msg.Error
 				}
+
 				if msg.Progress != nil {
-					glog.V(3).Infof("pushing image %s: %s", name, msg.Progress.String())
+					if msg.Progress.Current != 0 {
+						glog.V(4).Infof("pushing image %s: %s", name, msg.Progress.String())
```
Comment (runzexia) [inline pkg/docker/docker.go]: should glog.V(0)

File: pkg/scm/downloaders/git/clone.go [hunk 3]
```diff
@@ -23,16 +23,16 @@ func (c *Clone) Download(config *api.Config) (*git.SourceInfo, error) {
...
-		ref = "HEAD"
+	RevisionId := config.RevisionId
+	if RevisionId == "" {
+		RevisionId = "head"
 	}
 
 	if len(config.ContextDir) > 0 {
 		targetSourceDir = filepath.Join(config.WorkingDir, constants.ContextTmp)
-		glog.V(1).Infof("Downloading %q (%q) ...", config.Source, config.ContextDir)
+		glog.V(9).Infof("Downloading %q (%q) ...", config.Source, config.ContextDir)
```
Comment (runzexia) [inline pkg/scm/downloaders/git/clone.go]: should glog.V(0)

File: pkg/scm/downloaders/git/clone.go [hunk 4]
```diff
@@ -23,16 +23,16 @@ func (c *Clone) Download(config *api.Config) (*git.SourceInfo, error) {
...
 	} else {
-		glog.V(1).Infof("Downloading %q ...", config.Source)
+		glog.V(9).Infof("Downloading %q ...", config.Source)
```
Comment (runzexia) [inline pkg/scm/downloaders/git/clone.go]: should glog.V(0)

File: pkg/scm/downloaders/git/clone.go [hunk 5]
```diff
@@ -41,24 +41,25 @@ func (c *Clone) Download(config *api.Config) (*git.SourceInfo, error) {
...
 	}
 
-	err = c.Checkout(targetSourceDir, ref)
+	err = c.Checkout(targetSourceDir, RevisionId)
 	if err != nil {
 		return nil, err
 	}
-	glog.V(1).Infof("Checked out %q", ref)
+
+	glog.V(1).Infof("Checked out to %q", RevisionId)
```
Comment (runzexia) [inline pkg/scm/downloaders/git/clone.go]: should glog.V(0)

File: pkg/scm/downloaders/git/clone.go [hunk 6]
```diff
@@ -41,24 +41,25 @@ func (c *Clone) Download(config *api.Config) (*git.SourceInfo, error) {
...
 	if !config.IgnoreSubmodules {
 		err = c.SubmoduleUpdate(targetSourceDir, true, true)
 		if err != nil {
 			return nil, err
 		}
-		glog.V(1).Infof("Updated submodules for %q", ref)
+		glog.V(1).Infof("Updated submodules for %q", RevisionId)
```
Comment (runzexia) [inline pkg/scm/downloaders/git/clone.go]: should glog.V(0)

Comment (soulseen) [reply pkg/scm/downloaders/git/clone.go]: done

**Comment by soulseen** [comment]
@runzexia

**Pull request merged**